tool/site/client/src/fhir/helpers.ts

```diff
@@ -114,12 +114,15 @@ export function updateBundleIds(bundle: fhir.Bundle): void {
 
 export function updateValidityPeriod(bundle: fhir.Bundle): void {
   const start = convertMomentToISODate(moment.utc())
-  const end = convertMomentToISODate(moment.utc().add(1, "month"))
   getMedicationRequestResources(bundle).forEach(medicationRequest => {
     const validityPeriod = medicationRequest.dispenseRequest?.validityPeriod
     if (validityPeriod) {
-      validityPeriod.start = start
-      validityPeriod.end = end
+      const oldStart = moment.utc(medicationRequest.dispenseRequest.validityPeriod.start)
```

Comment:
`const oldStart = moment.utc(validityPeriod.start)`
already have a variable for the above